src/components/Modal/README.md

```diff
@@ -0,0 +1,1036 @@
+# Modal
+
+Modal allows prompting users to take or complete an action.
+
+## Basic Usage
+
+To implement the Modal component, you need to import it first:
+
+```js
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalTitle,
+} from '@react-ui-org/react-ui';
+```
+
+And use it:
+
+```docoff-react-preview
+React.createElement(() => {
```

Comment:
The modal in the examples get hidden behind the code area when scrolling.

![Screenshot from 2023-03-29 13-22-29](https://user-images.githubusercontent.com/4589176/228519989-e326b3a2-73cc-4878-adeb-40d74a430296.png)

Could @adamkudrna please take a look at it?

Reply:
Resolved in Docoff: https://github.com/react-ui-org/docoff/pull/13

Reply:
I have updated the docoff version and it is fixed now